pkg/api/openshiftcluster_validatestatic_test.go

```diff
@@ -52,21 +37,15 @@ func validOpenShiftCluster() *OpenShiftCluster {
 	return oc
 }
 
-func runTests(t *testing.T, tests []*validateTest, delta bool) {
+func runTests(t *testing.T, tests []*validateTest) {
 	for _, tt := range tests {
 		t.Run(tt.name, func(t *testing.T) {
 			oc := validOpenShiftCluster()
 			if tt.modify != nil {
 				tt.modify(oc)
 			}
 
-			current := &api.OpenShiftCluster{}
-			if delta {
```

Comment:
Deltas are not tested in the specific versions + underlying `immutable` package has own tests.